vmm/src/vstate.rs

```diff
@@ -584,8 +584,8 @@ mod tests {
             .expect("Cannot get supported cpuid");
         assert_eq!(
```

Comment:
I don't think we have to test CpuId::clone anymore since it's tested in the `FamStructWrapper` implementation.

Reply:
done.